tasks/prebootstrap.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Set history timestamps and size
+  blockinfile:
+    path: /etc/bash.bashrc
+    block: |
+      HISTSIZE=5000
+      HISTFILESIZE=3000
+      HISTTIMEFORMAT='%c%t'
```

Comment:
@speedlight I noticed this task is part of `prebootstrap.yml ` which is meant to be invoked only once, but what if the user wants to change a bash history setting?

Reply:
@sestrella I'm not sure about caring about especific user configuration as we talked about freeing sb-debian-base from things like user configurations, I think that could go in a per project playbooks as we done with aliases. As this basic configuration is more security common best practice than a user preference I thought preboostrap is a good place to put it.